common/commands/src/coconut/generate_freepass.rs

```diff
@@ -55,17 +59,17 @@ pub struct Args {
 
 async fn get_freepass(
     api_clients: Vec<CoconutApiClient>,
-    aggregate_vk: &VerificationKey,
+    aggregate_vk: &VerificationKeyAuth,
     threshold: u64,
     epoch_id: EpochId,
     signing_account: &AccountData,
-    expiration_date: OffsetDateTime,
+    expiration_date_ts: u64,
```

Comment:
why not keep the proper timestamp given that's what our CLIs and APIs will be accepting? in particular the vpn api

Reply:
TO FIX : timestamps

Reply:
Fixed in f2cfee9ee8605d3cc8dd3bf25a119aa38aa37acd